numpy/core/defchararray.py

```diff
@@ -275,6 +275,15 @@ def str_len(a):
     See Also
     --------
     builtins.len
+
+    Examples
+    --------
+    >>> a = "example"
+    >>> np.char.str_len(a)
+    array(7)
+    >>> a = "hello world"
+    >>> np.char.str_len(a)
+    array(11)
```

Comment:
Can reformat this similar to the `isdigit` one?  What I mean is write `a = np.array()` with multiple entries rather than showing two single strings as an example, show one array with 3 entries that have different lengths.

Using an array is probably more instructional and also more concise.

Reply:
Please review the updated examples.